test/unit/api_spec.js

```diff
@@ -583,12 +618,15 @@ describe('api', function() {
     var pdf2 = combineUrl(window.location.href, '../pdfs/TAMReview.pdf');
     // A PDF using the Arial font.
     var pdf3 = combineUrl(window.location.href, '../pdfs/issue6068.pdf');
+    var loadingTasks = [];
```

Comment:
Isn't the `loadingTasks` and the `pdfDocuments` equivalent here (i.e. I don't think you'll need both)?